.github/workflows/wheels-pure-build.yml

```diff
@@ -67,17 +79,15 @@ jobs:
         ctk: ${{ matrix.ctk }}
         package-name: ${{ inputs.package-name }}
 
+    - name: Run before-wheel command
+      run: "${{ inputs.before-wheel }}"
+
     - name: Build pure python wheel with pip wheel
       run: |
         versioneer_override="$(rapids-pip-wheel-version ${{ needs.wheel-epoch-timestamp.outputs.rapids_epoch_timestamp }})"
         export RAPIDS_PY_WHEEL_VERSIONEER_OVERRIDE="${versioneer_override}"
-        python -m pip wheel -w ./dist ${{ inputs.package-dir }} --no-deps
+        python -m pip wheel -w ./dist ${{ inputs.package-dir }} --no-deps -vvv
```

Comment:
Are the `-vvv` flags intended to be added in this PR?

Reply:
Yes, just putting it on par with the verbosity of cibuildwheel (where `CIBW_BUILD_VERBOSITY_LINUX: 3` results in `pip wheel -vvv`).